### PR TITLE
[Fixes #108813328] Turn on debug stream when terminating error is thrown

### PR DESCRIPTION
**Pull request by jianghaolu**

### CODE REVIEW

File: src/Common/Commands.Common/AzurePSCmdlet.cs [hunk 1]
```diff
@@ -497,6 +503,11 @@ protected virtual void Dispose(bool disposing)
 
         public void Dispose()
         {
+            try
+            {
+                FlushDebugMessages();
+            }
+            catch (Exception) { }
```
Comment (markcowl) [inline src/Common/Commands.Common/AzurePSCmdlet.cs]: You can just use ```catch{}```

**Comment by markcowl** [comment]
One nit.  Can you please add a test to the scripts file that will produce Debug output and throw an error?

Otherwise, LGTM

**Comment by markcowl** [comment]
on demand run here:  http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/349/

**Comment by jianghaolu** [comment]
Addressed. New on-demand: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/353/

**Comment by markcowl** [comment]
@jianghaolu Still on-demand failures, please fix

**Comment by jianghaolu** [comment]
new on-demand: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/366/

**Pull request merged**